versions/3.0.md

```diff
@@ -1945,7 +1945,7 @@ An object to hold data types that can be consumed and produced by operations. Th
 
 Field Pattern | Type | Description
 ---|:---:|---
-<a name="definitionsName"></a>{name} | [Schema Object](#schemaObject) | A single definition, mapping a "name" to the schema it defines.
+<a name="definitionsName"></a>{name} | [Schema Object](#schemaObject) | A single definition, mapping a "name" to the schema it defines.  The following characters are legal in the `definitionsName`: `a-zA-Z0-9.-_()[]\`.  All other characters, including high-level characters, are not legal in the `definitionsName`.  The name may be used to namespace the definitions--it is the tooling vendor's job to ensure that the namespace is honored.
```

Comment:
@fehguy  Just a few thoughts:

1. "high level characters" could be potentially misinterpreted. For example, if I google "high level characters", I get this https://en.wikipedia.org/wiki/Epic_level. Perhaps "special characters", which is in more common use (and what I think you meant).
2. An example of what a namespace might look like would be useful here for someone trying to understand what namespace means in this context.